fire-cracker/SendIT-react
# #164684154 Implement user can login/signup using react

#### What does this PR do?
- Implements Login and Sign-up components
- Implements Login and Sign-up functionality using actions and reducers

#### How should this be manually tested?
- Clone repo
- Check out to branch `feature/164684154/Login-Signup`
- Run application with `npm run start-dev` in the terminal
- Visit `http://localhost:8080/`
- Click on `login` or `Sign Up` buttons.
- Fill the Sign-up form and verify from your account

#### What are the relevant pivotal tracker stories?
- [Implement user can login/signup using react](https://www.pivotaltracker.com/story/show/164684154)
